### PR TITLE
Add tutorials filtering by topics

### PR DESCRIPTION
**Pull request by tbille**
## Done

- Add dropdown to filter tutorials per topic
- For now I use the current topics that we have on discourse, we can add an updated version of those in a later iteration.

## QA

- `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8041/tutorials
- Play with the filters, make sure you have results for all of them.

### CODE REVIEW

File: static/js/tutorials-list.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+(function () {
```
Comment (solazio) [inline static/js/tutorials-list.js:1]:
```suggestion
(function () {
  const selector = "tutorials-topic";
  const topicsFilter = document.getElementById(selector);

  if (topicsFilter) {
    topicsFilter.addEventListener("change", (e) => {
      window.location.search = `?topic=${e.target.value}`;
    });
  } else {
    throw new Error(`${selector} is not a valid element!`);
  }
})();
```

**Review by solazio — APPROVED** [review]
It works fine as it is, but I would change the `js` file.

File: static/js/tutorials-list.js (hunk 1 above)
Comment (tbille) [reply static/js/tutorials-list.js:1]: I like it.

**Review by solazio — APPROVED** [review]
Looks good :+1:. Thanks

**Pull request merged**